src/api/v1/endpoint/chat.py

```
@@ -1,14 +1,12 @@
-)  # 👈 ChatResult 스키마 가져오기
-from src.services.rag_service import generate_answer  # 👈 RAG 서비스 함수 가져오기
-import logging
+from src.core.database import get_db
+from src.schemas.common import CommonResponse
+from src.schemas.chat import ChatRequest, ChatResult
+from src.schemas.compare import CompareRequest, CompareResult
+from src.services.chat_service import generate_answer
+from src.services.compare_service import compare_laws
 
 router = APIRouter()
 logger = logging.getLogger(__name__)
```

Ruff (0.15.1)

[error] Undefined name `logging`

(F821)